newIDE/app/src/ResourcesList/ResourceUtils.js

```diff
@@ -8,3 +10,9 @@ export const createOrUpdateResource = (project, gdResource, resourceName) => {
   resourcesManager.addResource(gdResource);
   gdResource.delete();
 };
+
+export const getLocalResourceFullPath = (project, resourceName) => {
+  let resourcePath = ResourcesLoader.getResourceFullUrl(project, resourceName);
+  resourcePath = resourcePath.substring(7, resourcePath.lastIndexOf('?cache='));
```

Comment:
Excellent 👍
I wonder if at some point this should be refactored in ResourcesLoader. For the time being, this is great to have it in a single function. Will be easy to refactor again later if needed.